backend/de.metas.business/src/main/java/de/metas/invoice/process/C_Invoice_MassDiscountOrWriteOff.java

```diff
@@ -118,7 +118,7 @@ protected String doIt()
 	private void invoiceDiscount(@NonNull final I_C_Invoice invoice)
 	{
 		final CurrencyId currencyId = CurrencyId.ofRepoId(invoice.getC_Currency_ID());
-		final Money invoiceOpenAmt = Money.of(allocationDAO.retrieveOpenAmt(invoice, true), currencyId);
+		final Money invoiceOpenAmt = Money.of(allocationDAO.retrieveOpenAmtInInvoiceCurrency(invoice, true), currencyId);
```

Comment:
IAllocationDAO.retrieveOpenAmt is doing the same, i.e. returns Money.
Why not using it?